python_benchmarks/qr_codes/README.md

```diff
@@ -0,0 +1,2 @@
+Download boofcv dataset (https://boofcv.org/notwiki/regression/fiducial/qrcodes_v3.zip) to this folder.
+Install opencv-python.
```

Comment:
Need to define how to point to WeChat models location.

Reply:
added info to README